### ISSUE TITLE
Sonarr Background Elements not Rendered

### ISSUE THREAD

**Issue by Ryonez** [GilbN/theme.park#219]
**Describe the bug**
The background/backdrop for various elements do not render. 

**Theme Install Method** (Don't skip)
- Webserver subfilter Config

        proxy_set_header Accept-Encoding "";
        sub_filter
                '</head>'
                        '<link rel="stylesheet" type="text/css" href="https://gilbn.github.io/theme.park/CSS/themes/sonarr/plex.css">
                </head>';
        sub_filter_once on;

**Application Info** (Don't skip)
- Sonarr - 3.0.6.1265
- Stable
- binhex/arch-sonarr

**To Reproduce**
Open something that would use a backdrop in the element.

**Expected behavior**
For there to be backdrops.

**Screenshots**
![image](https://user-images.githubusercontent.com/16157130/123521199-5d01ab00-d709-11eb-9450-6946fdf1e0e6.png)
![image](https://user-images.githubusercontent.com/16157130/123521208-6ee34e00-d709-11eb-930c-29de6eb0e597.png)
![image](https://user-images.githubusercontent.com/16157130/123521234-802c5a80-d709-11eb-98ec-7dbc4ad139d7.png)

**Desktop (If applicable) (please complete the following information):**
 - Browser - Vivaldi 4.0.2312.33 (Stable channel) (64-bit)

**Additional context**
I noticed a difference between Vivaldi and Firefox.

Vivaldi:
![image](https://user-images.githubusercontent.com/16157130/123521357-490a7900-d70a-11eb-8938-f01664caf9ee.png)
Firefox:
![image](https://user-images.githubusercontent.com/16157130/123521372-532c7780-d70a-11eb-9869-1e3f7dadb51c.png)

I'm not sure if Firefox is displaying this correctly either, but is does have backgrounds for all of the elements at least.

**Comment by GilbN** [comment]
Works on Vivaldi here. 
Vivaldi is Chromium so it should work just fine. 
![image](https://user-images.githubusercontent.com/24592972/123523877-19864d00-d6c7-11eb-924d-90400fa573cb.png)
![image](https://user-images.githubusercontent.com/24592972/123523915-4fc3cc80-d6c7-11eb-8ad2-02fdd91d7749.png)
![image](https://user-images.githubusercontent.com/24592972/123523958-90234a80-d6c7-11eb-8195-7b7e564a9fa9.png)

**Comment by Ryonez** [comment]
Now I feel like an idiot. I checked my extension to see if something was blocking it but couldn't find anything.
Tried it in a private window and it all rendered correctly. 
Turns out at some point the Dark Reader extension activated on a few of my sites.
Heck, the theme of radarr worked despite finding it was also being effected. The changes were just much more subtle.

Sorry for the trouble, and thank you for having a look on your end.
